pkg/client/account.go

```diff
@@ -30,11 +35,44 @@ func (c *Client) CheckBalance(address string) (types.Coins, error) {
 	return resp.GetBalances(), nil
 }
 
+//GetAccount fetches on-chain account details from the Sonr Network
+func (c *Client) GetAccount(address string) (*acctypes.BaseAccount, error) {
+	// Create a connection to the gRPC server.
+	grpcConn, err := grpc.Dial(
+		c.GetRPCAddress(),   // Or your gRPC server address.
+		grpc.WithInsecure(), // The Cosmos SDK doesn't support any transport security mechanism.
+	)
+	defer grpcConn.Close()
+	if err != nil {
+		return nil, err
+	}
+	encodingConfig := app.MakeEncodingConfig()
+
+	initClientCtx := client.Context{}.
+		WithCodec(encodingConfig.Marshaler).
+		WithInterfaceRegistry(encodingConfig.InterfaceRegistry).
+		WithTxConfig(encodingConfig.TxConfig).
+		WithLegacyAmino(encodingConfig.Amino)
+
+	queryCli := acctypes.NewQueryClient(grpcConn)
+	resp, err := queryCli.Account(context.Background(), &acctypes.QueryAccountRequest{Address: address})
+	if err != nil {
+		return nil, err
+	}
+	accBz, err := initClientCtx.Codec.MarshalJSON(resp.Account)
+	if err != nil {
+		return nil, err
+	}
+	var acc acctypes.BaseAccount
+	initClientCtx.Codec.UnmarshalJSON(accBz, &acc)
+	return &acc, nil
+}
+
 // RequestFaucet requests a faucet from the Sonr network
 func (c *Client) RequestFaucet(address string) error {
 	values := faucetRequest{
 		Address: address,
-		Coins:   []string{"12000snr"},
+		Coins:   []string{"120snr"},
```

Comment:
Can you verify that the motor unit tests still work with this change? I believe this was bumped up as a workaround for something...